helm/grafana/values.yaml

```diff
@@ -1,16 +1,32 @@
 global:
+  # -- Overrides the Docker registry globally for all images
   imageRegistry: null
+
+  # To help compatibility with other charts which use global.imagePullSecrets.
+  # Allow either an array of {name: pullSecret} maps (k8s-style), or an array of strings (more common helm-style).
+  # Can be templated.
+  # global:
+  #   imagePullSecrets:
+  #   - name: pullSecret1
+  #   - name: pullSecret2
+  # or
+  # global:
+  #   imagePullSecrets:
+  #   - pullSecret1
+  #   - pullSecret2
   imagePullSecrets: []
 
+
 ciliumNetworkPolicy:
   enabled: true
 
 grafana:
   rbac:
     create: true
     ## Use an existing ClusterRole/Role (depending on rbac.namespaced false/true)
-    # useExistingRole: name-of-some-(cluster)role
-    pspEnabled: true
+    # useExistingRole: name-of-some-role
+    # useExistingClusterRole: name-of-some-clusterRole
+    pspEnabled: false
```

Comment:
Why do you want to disable PSPs .

Reply:
We do not use them anymore

Reply:
Oh yeah my bad we're using the PSS :facepalm: 
I took one for the other